cmd/runtime/main.go

```diff
@@ -26,10 +27,14 @@ import (
 	"github.com/cosi-project/runtime/pkg/state/protobuf/server"
 )
 
-var socketPath string
+var (
+	addressAndPort string
+	socketPath     string
+)
 
 func main() {
-	flag.StringVar(&socketPath, "socket-path", "/var/run/cosi-runtime.sock", "path to the UNIX socket to listen on")
+	flag.StringVar(&socketPath, "socket-path", "", "path to the UNIX socket to listen on")
```

Comment:
this changes default behavior by making runtime listen on TCP by default vs. file socket

I think having it listening on `*:50000` by default is not a good default

Reply:
Default to unix socket? Default to `127.0.0.1`? Either way is fine by me.

Reply:
I would keep it as file socket, not saying it's more secure than localhost, but at least shows the intention better as the API doesn't have any auth at the moment.